app/views/content/landing/how-to-become-a-teacher/_content.html.erb

```diff
@@ -0,0 +1,57 @@
+<div class="row">
+  <section class="col col-720">
+    <h2 class="heading--box-blue">What qualifications do I need?</h2>
+
+    <div class="inset">
+      <p>
+        You'll need to train for qualified teacher status (QTS) to be able to teach in most primary and secondary schools in England.
+      </p>
+
+      <p>
+        If you have a degree or equivalent qualification, you can do postgraduate primary or secondary teacher training.
+        If you do not have a degree and are not studying for one, you can do undergraduate teacher training.
+      </p>
+
+      <p>
+        <%= link_to("Check the qualifications you need to teach", page_path("is-teaching-right-for-me/qualifications-you-need-to-teach")) %>.
+    </div>
+  </section>
+
+  <section class="col col-720">
+    <h2 class="heading--box-blue">Can I get help funding my training?</h2>
+
+    <div class="inset">
+      <p>
+        Undergraduate and postgraduate teacher training course fees are around £9,250.
```

Comment:
```suggestion
        Undergraduate and postgraduate teacher training course fees are around £9,250 per year.
```
Someone pointed this out in UR actually that we might need to be clearer it's per year, not for the whole course